sccp/imports/CoH.py

```diff
@@ -164,57 +126,6 @@ def CoH_xarray(numCells, cond, allmarkers):
     "BC-14",
 ]
 
-cell_types = [
-    "T",
-    "CD16 NK",
-    "CD8+",
-    "CD4+",
-    "CD4-/CD8-",
-    "Treg",
-    "Treg 1",
-    "Treg 2",
-    "Treg 3",
-    "CD8 TEM",
-    "CD8 TCM",
-    "CD8 Naive",
-    "CD8 TEMRA",
-    "CD4 TEM",
-    "CD4 TCM",
-    "CD4 Naive",
-    "CD4 TEMRA",
-    "CD20 B",
-    "CD20 B Naive",
-    "CD20 B Memory",
-    "CD33 Myeloid",
-    "Classical Monocyte",
-    "NC Monocyte",
-]
-
-cell_types_rename = {
-    "T": 0,
-    "CD16 NK": 1,
-    "CD8+": 2,
-    "CD4+": 3,
-    "CD4-/CD8-": 4,
-    "Treg": 5,
-    "Treg 1": 6,
-    "Treg 2": 7,
-    "Treg 3": 8,
-    "CD8 TEM": 9,
-    "CD8 TCM": 10,
-    "CD8 Naive": 11,
-    "CD8 TEMRA": 12,
-    "CD4 TEM": 13,
-    "CD4 TCM": 14,
-    "CD4 Naive": 15,
-    "CD4 TEMRA": 16,
-    "CD20 B": 17,
-    "CD20 B Naive": 18,
-    "CD20 B Memory": 19,
-    "CD33 Myeloid": 20,
-    "Classical Monocyte": 21,
-    "NC Monocyte": 22,
-}
 
 marker_dict_all = [
```

Comment:
@aarmey Should I be using all markers in the flow cytometry, or only pSTAT and surface markers?

Reply:
Given that scRNAseq essentially includes all the markers, and has worked best so far for Pf2, let's include them all.